docs/migrating.md

```diff
@@ -0,0 +1,258 @@
+# Migrating from `react-stripe-elements`
+
+This guide includes of step by step instructions for migrating your Stripe
+integration from
+[`react-stripe-elements`](https://github.com/stripe/react-stripe-elements) to
```

Comment:
Should we refer to the library as "React Stripe" or `react-stripe-elements`? The usage seems mixed here and in the README.

Reply:
The old library is `react-stripe-elements` the new one is React Stripe

Reply:
Ah whoops, I meant to point out that we are referring to the new library as both "React Stripe" and `@stripe/react-stripe`. Seems like we could just use one or the other (I prefer "React Stripe").

Reply:
Ah. Good point. Fixed.